packages/dom/src/range/match.ts

```diff
@@ -18,22 +18,21 @@
  * under the License.
  */
 
-import type { RangeSelector, Selector } from '@annotator/selector';
-
-import { ownerDocument } from '../scope';
-import type { DomMatcher, DomScope } from '../types';
```

Comment:
If we stop using them, we’d better remove the type file, and the scope file too.